### ISSUE TITLE
Mesh index dose not updated, when node skipped.

### ISSUE THREAD

**Issue by weepingtown** [donalffons/opencascade.js#95]
**version:** 2.0.0-beta.45aa6c6

When a node without triangulation data exists in the .stp file, the RWGltf_CafWriter will skip this node. However, in the generated .gltf file, the mesh index of this node is not updated accordingly. As a result, the gltf file cannot be parsed properly.
I used Open Cascade 7.6 to create a C++ demo program for verification and found that the problem did not recur. So I guess it's a problem specific to opencascade.js. I'll submit an issue later to explain this in detail.

1. the js code as below:
```js
    let shape = openCascade.XCAFDoc_ShapeTool.GetShape_2(rootLabels.First())
    let meshAlgo = new openCascade.BRepMesh_IncrementalMesh_2(shape, 2, openCascade.false, 1, openCascade.true);
    meshAlgo.Perform_1(new openCascade.Message_ProgressRange_1());

    //output gltf
    let fileInfo = new openCascade.TColStd_IndexedDataMapOfStringString_1();
    fileInfo.Add(new openCascade.TCollection_AsciiString_2("Author"), new openCascade.TCollection_AsciiString_2("glTf Converter"));
    fileInfo.Add(new openCascade.TCollection_AsciiString_2("Comments"), new openCascade.TCollection_AsciiString_2("version 0.01"));

    let gltfWriter = new openCascade.RWGltf_CafWriter(new openCascade.TCollection_AsciiString_2("temp.glb"), true);
```
error in DevTools console
![image](https://user-images.githubusercontent.com/6825667/148767575-1d3bfcdb-fe2a-43ed-9800-8117f251f24b.png)

2. the c++ code as below:
```c++
    TColStd_IndexedDataMapOfStringString aFileInfo;
    aFileInfo.Add("Author", "glTf Converter");
    aFileInfo.Add("Comments", "version 0.01");
    // write or export the document
    TColStd_IndexedDataMapOfStringString aMetadata;
    RWGltf_CafWriter aGltfWriter(fileName, isBinary);
    // STEP reader translates into mm units by default
    aGltfWriter.ChangeCoordinateSystemConverter().SetInputLengthUnit(0.001);
    aGltfWriter.ChangeCoordinateSystemConverter().SetInputCoordinateSystem(RWMesh_CoordinateSystem_Zup);
 
    //Handle(Message_ProgressIndicator) process;
    Message_ProgressScope process;
    if (!aGltfWriter.Perform(docment, aFileInfo, process.Next()))
    {
        // export error
    }
```

3. c++ demo output in console

```
$ ./converter.exe -i as1_pe_203.stp -o as1_pe_203.gltf
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
RWGltf_CafWriter skipped node 'COMPOUND' without triangulation data
```
4. gltf file comparison
![image](https://user-images.githubusercontent.com/6825667/148767880-c3c32cea-2412-4dd8-b727-e1922ad81e41.png)

5. the stp asset file:
[input.zip](https://github.com/donalffons/opencascade.js/files/7838874/input.zip)

**Comment by weepingtown** [comment]
I notice that opencascade.js uses occt 7.5. 
```
ENV OCCT_COMMIT_HASH_FULL fecb042498514186bd37fa621cdcf09eb61899a3
ENV OCCT_COMMIT_HASH fecb042
WORKDIR /occt/
RUN \
  curl "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=${OCCT_COMMIT_HASH_FULL};sf=tgz" -o occt.tar.gz && \
  tar -xvf occt.tar.gz
```
<img width="304" alt="image" src="https://user-images.githubusercontent.com/6825667/148798312-2d0475ab-5ee1-4dfe-b4cf-ac48ea13366c.png">

And also, in the commit log of OCCT, I found that this issue has been fixed and released in version 7.6. The code is as follows:
<img width="1585" alt="image" src="https://user-images.githubusercontent.com/6825667/148797578-41fdebce-bc55-41d0-960c-630d84a2dfd5.png">
issue info:
```
0032527: Data Exchange, RWGltf_CafWriter - make name format configurable
Added properties RWGltf_CafWriter::NodeNameFormat() and ::MeshNameFormat().
```

**Comment by weepingtown** [comment]
merge request: #96
